plugins/modules/vmware_drs_group.py

```diff
@@ -62,6 +62,19 @@
     required: false
     type: list
     elements: str
+  operation:
```

Comment:
The UNIX group management is done a bit differently, you've got the `group` module that can create/delete group.
And in the `user` group, you can use the `groups` key to define or break a group association. The `append` key allow adjust the behaviour a bit:
- If yes, add the user to the groups specified in groups.
- If no, user will only be added to the groups specified in groups, removing them from all other groups.

I would prefer we take this approach because it's more logical and easier to document.

Reply:
Do you mean a separate module for managing the VMs/hosts in the group?